src/__tests__/fireEvent.test.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import {
+  View,
+  TouchableOpacity,
+  Text,
+  ScrollView,
+  TextInput,
+} from 'react-native'; // eslint-disable-line import/no-unresolved
+import fireEvent from '../fireEvent';
+import { render } from '..';
+
+const OnPressComponent = ({ onPress }) => (
+  <View>
+    <TouchableOpacity onPress={onPress} testID="button">
+      <Text testID="text-button">Press me</Text>
+    </TouchableOpacity>
+  </View>
+);
+
+const WithoutEventComponent = () => (
+  <View>
+    <Text testID="text">Content</Text>
+  </View>
+);
+
+const CustomEventComponent = ({ onCustomEvent }) => (
+  <TouchableOpacity onPress={onCustomEvent}>
+    <Text>Click me</Text>
+  </TouchableOpacity>
+);
+
+jest.mock(
+  'react-native',
+  () => ({
+    View(props) {
+      return props.children;
+    },
+    ScrollView(props) {
+      return props.children;
+    },
+    Text(props) {
+      return props.children;
+    },
+    TextInput() {
+      return null;
+    },
+    TouchableOpacity(props) {
+      return props.children;
+    },
+  }),
+  { virtual: true }
+);
+
+describe('fireEvent.invokeEvent', () => {
+  test('should invoke specified event', () => {
+    const onPressMock = jest.fn();
+    const { getByTestId } = render(<OnPressComponent onPress={onPressMock} />);
+
+    fireEvent.invokeEvent(getByTestId('button'), 'press');
```

Comment:
can we go with `fireEvent(getByTestId('button'), 'press');` to be on par with react-testing-library?